tracex_project/tracex/logic/utils.py

```diff
@@ -178,6 +178,24 @@ def create_xes(csv_file, name, key):
             start_timestamp_key="start_timestamp",
             timestamp_key="time:timestamp",
         )
+
+        return str(output_path / output_name)
+
+    @staticmethod
+    def create_xes_from_df(df, name, key):
+        """Creates a xes with all traces from the regarding df."""
+        dataframe = Conversion.prepare_df_for_xes_conversion(df, key)
+
+        output_name = name + "_" + key + ".xes"
+        pm4py.write_xes(
+            dataframe,
+            (output_path / output_name),
+            case_id_key="case:concept:name",
+            activity_key="concept:name",
```

Comment:
Is this naming because of legcay or because someone knows, what this stands for?

Reply:
That was a necessary change because not all rows of the trace were being shown. This was due to incorrect spelling. These lines fix that problem.